README.md

```diff
@@ -6,44 +6,61 @@
 
 # 📖 OSPOlogy: The Study of OSPOs
 
-The [TODO Group](https://todogroup.org) hosts a monthly meeting and newsletter to study and discuss the status of OSPOs.
+✨ **`OSPOlogy` hosts the study and open community communication to discuss the status of open source program offices across regions.**
 
-OSPOlogy is open to all, from folks just starting their OSPO journey to students to even seasoned OPSOers!
+## 🚀 Our Mission
 
-## Global Community Meeting (monthly)
+The [TODO Group](https://todogroup.org) is an open community of practitioners who aim to create and share knowledge, collaborate on practices, tools, and other ways to run successful and effective Open Source Program Offices or similar Open Source initiatives. TODO Group is formed by its [Community participants](https://todogroup.org/community) and [General Members](https://todogroup.org/members).
 
-Everyone is welcome to attend the monthly TODO Group OSPOlogy meeting: https://community.linuxfoundation.org/todo-group/
+## 🦺 Community
 
-## TODO EU Chapter Sync Meetings (Monthly)
+TODO has a community page with an onboarding list that shares the networking spaces and communication channels: https://todogroup.org/community/. To participate in OSPO related discussions you can [submit questions](https://github.com/todogruop/ospology/discussions) or join the [TODO Group Slack](https://slack.todogroup.org). 
 
-The TODO EU chapter was formed by TODO Members based in Europe. This chapter was created with the aim of working together to improve OSPO adoption and education within Europe and discuss with the broader community the challenges European organizations face when implementing an open source program.
+### 🔭 Global Community Meetings: OSPOlogy
 
-EMEA-friendly time meetings are scheduled every last Thursday of the month at 3PM (CET): https://community.linuxfoundation.org/todo-group-europe/
+The global and public monthly TODO Group and wider OSPO community's meetings. These meetings have featured OSPO-related topics with special guests and community discussions. All these meetings are recorded, edited, and recorder on Youtube OSPOlogy Channel, so everyone across the globe can have access to it via: https://github.com/todogroup/ospology/tree/main/meetings#event-page-and-past-sessions
```

Comment:
You may want the url in a hard link instead of having github generate a link and display the full URL. Your option.